docs/section12/Rule12-3.md

```diff
@@ -1,38 +1,58 @@
-
-**[Back](../_toc.md)**
+## Rule Logic:  
+- Iterate through the spaces in the proposed model: `for space_p in P_RMD...spaces`  
+  - Get the lighting space type: `space_type_p = space_p.lighting_space_type` 
+  - Iterate through the miscellaneous equipment loads in the space: `for misc_equip_p in space_p.miscellaneous_equipment:`  
+    - Get the proposed automatic receptacle control: `auto_receptacle_control_p = misc_equip_p.has_automatic_control`  
+    - If the space type is not in the list of space types where receptacle controls may be required, and the space has automatic receptacle controls installed: `if space_type_p not in EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES and auto_receptacle_control_p:`  
+      - Get the percentage of controlled receptacles: `controlled_percentage = misc_equip_p.automatic_controlled_percentage`
+      - Calculate the expected receptacle power credit: `expected_receptacle_power_credit = 0.10 * controlled_percentage`
+      - Get the associated miscellaneous equipment object in the baseline model: `misc_equip_b = get_component_by_id(misc_equip_p.id, B_RMD)`
+      - Get the baseline miscellaneous equipment multiplier schedule: `hourly_multiplier_schedule_b = misc_equip_b.multiplier_schedule`
+      - Get the proposed miscellaneous equipment multiplier schedule: `hourly_multiplier_schedule_p = misc_equip_p.multiplier_schedule`
+      - Create a list of expected hourly values based on the expected receptacle power credit: `expected_hourly_values = [hour_value * (1 - expected_receptacle_power_credit) for hour_value in hourly_multiplier_schedule_b.hourly_values]`
+      - If it is a leap year, set the schedule comparison mask to 8784 hours, else set it to 8760 hours: `mask_schedule = [1] * 8784 if is_leap_year else [1] * 8760`
+      - Compare the expected miscellaneous equipment load schedule to the proposed miscellaneous equipment load schedule: `credit_comparison_data = compare_schedules(expected_hourly_values, hourly_multiplier_schedule_p, mask_schedule)`
+      - Compare the baseline miscellaneous equipment load schedule to the proposed miscellaneous equipment load schedule: `no_credit_comparison_data = compare_schedules(hourly_multiplier_schedule_b, hourly_multiplier_schedule_p, mask_schedule)`
+
+
+**Rule Assertion:**  
+- Case 1: If the expected receptacle power credit was applied as expected for all hours: PASS `if credit_comparison_data['total_hours_matched'] == len(hourly_multiplier_schedule_p.hourly_values == len(expected_hourly_values): PASS` 
+- Case 2: If the expected receptacle power credit was expected, but baseline and proposed are identical: UNDETERMINED `if no_credit_comparison_data['total_hours_matched'] == len(hourly_multiplier_schedule_b.hourly_values == len(hourly_multiplier_schedule_p.hourly_values): UNDETERMINED`  
+- Case 3: Else, a credit or penalty was applied incorrectly: FAIL `else: outcome = FAIL`  
```

Comment:
So this would fail if it is one of the space types where it is expected and does not meet the 25 V, 15 and 20 amp criteria. Maybe we should have a separate fail for is it is one of the space types and is modeling a control credit, we could say "Fail unless controls are not required based on 90.1 Section 8.4.2 requirements. Based on the space type it appears that the requirements in 90.1 Section 8.4.2 would be applicable; however, there may be an applicable exception or the receptacles do not meet the 125 V, 15 and 20 amp criteria in the section. "

Reply:
This rule is not applicable if the space type is in the list of space types from 8.4.2